.kitchen.yml

```diff
@@ -1,6 +1,15 @@
 ---
+driver:
```

Comment:
if you put this in `~/.kitchen/config.yml` instead you won't need this in the rest of your kitchen file.  

I'm actually working on setting up CI servers to spin up instances on the cloud. I prefer not having this here so I don't have to populate my CI jobs with `.kitchen.local.yml` files